src/Containers/Synchronization/Index.js

```diff
@@ -36,7 +40,7 @@ const IndexSynchronizationContainer = () => {
   useEffect(async () => {
     const result = await GetNonSynchronizedService()
     setUnsynced(result)
-  }, [])
+  }, [syncLoading])
```

Comment:
You sure about this ? Il va le faire deux fois non ? Une fois quand syncLoading passe a true et une fois quand ca passe a false ?